package.json

```diff
@@ -41,9 +41,14 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run prettier"
+      "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "./src/**/*.{tsx,ts,js}": [
```

Comment:
will this not do the server files in /server?

Reply:
Also I think prettier runs on css and json and maybe some other file types too.

Reply:
True, will add them to the list!